includes/wc-product-functions.php

```diff
@@ -1442,6 +1443,16 @@ function wc_update_product_lookup_tables_column( $column ) {
 		case 'downloadable':
 		case 'virtual':
 			$column = esc_sql( $column );
+
+			$wpdb->query(
```

Comment:
Is there no way we can rather than running an additional update just do one based on if virtual or not?

Reply:
Comment below. Is there a better way?